pkg/cmd/status.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/testground/testground/pkg/api"
+	"github.com/testground/testground/pkg/client"
+	"github.com/urfave/cli/v2"
+)
+
+var StatusCommand = cli.Command{
+	Name:      "status",
+	Usage:     "get the current status for a certain task",
+	Action:    statusCommand,
+	ArgsUsage: "[task_id]",
+	Flags:     []cli.Flag{},
+}
+
+func statusCommand(c *cli.Context) error {
+	ctx, cancel := context.WithCancel(ProcessContext())
+	defer cancel()
+
+	if c.NArg() != 1 {
+		return errors.New("missing run id")
+	}
+
+	var (
+		id = c.Args().First()
+	)
+
+	cl, _, err := setupClient(c)
+	if err != nil {
+		return err
+	}
+
+	r, err := cl.TaskStatus(ctx, &api.TaskStatusRequest{ID: id})
+	if err != nil {
+		return err
+	}
+	defer r.Close()
+
+	res, err := client.ParseTaskStatusResponse(r)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("ID:\t\t%s\n", res.ID)
+	fmt.Printf("Created:\t%s\n", res.Created)
```

Comment:
This value appears to be set wrong. When I submit a job, I get the following status back:

```
ID:             b513cdd1-0153-4b6b-be87-4ddbbbd0c08f
Created:        4189-12-26 04:53:15.7134801 +0000 UTC
Type:           run
Status:         processing
Last update:    2020-08-17 13:50:47.2888156 +0000 UTC
```